src/package.xml

```diff
@@ -995,6 +995,8 @@
         <members>DataImport__c.Contact1Imported__c</members>
```

Comment:
I think the new CMT records are all missing from package.xml (Data_Import_Field_Mapping.Payment_Authorized_Date, etc)